upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```
@@ -1526,10 +1526,10 @@ spec:
               port: http
           resources:
             limits:
-              memory: 1Gi
+              memory: {{ or .Karpenter.MemoryLimit "1Gi" }}
             requests:
-              cpu: 500m
-              memory: 1Gi
+              cpu: {{ or .Karpenter.CPURequest "25m" }}
+              memory: {{ or .Karpenter.MemoryRequest "200Mi" }}
```

Let's be more conservative for now.
```suggestion
              cpu: {{ or .Karpenter.CPURequest "200m" }}
              memory: {{ or .Karpenter.MemoryRequest "500Mi" }}
```